docs/source/examples/mg-mode-part-iii.ipynb

```diff
@@ -0,0 +1,1460 @@
+   "metadata": {
+    "collapsed": false
+   },
+   "outputs": [],
+   "source": [
+    "# Load the isotropic statepoint file\n",
+    "iso_mgsp = openmc.StatePoint(iso_mg_spfile, autolink=False)\n",
+    "iso_mgsum = openmc.Summary(iso_mg_sumfile)\n",
+    "iso_mgsp.link_with_summary(iso_mgsum)\n",
+    "\n",
+    "# Load the angle-dependent statepoint file\n",
+    "angle_mgsp = openmc.StatePoint('statepoint.' + str(batches) + '.h5')"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Eigenvalue Comparison\n",
+    "Next, we can load the eigenvalues for comparison and do that comparison"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 38,
+   "metadata": {
+    "collapsed": false
+   },
+   "outputs": [],
+   "source": [
+    "ce_keff = sp.k_combined\n",
+    "iso_mg_keff = iso_mgsp.k_combined\n",
+    "angle_mg_keff = angle_mgsp.k_combined\n",
+    "\n",
+    "# Find eigenvalue bias\n",
+    "iso_bias = 1.0E5 * (ce_keff[0] - iso_mg_keff[0])\n",
+    "angle_bias = 1.0E5 * (ce_keff[0] - angle_mg_keff[0])"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Lets compare the eigenvalues in units of pcm"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 39,
+   "metadata": {
+    "collapsed": false
+   },
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "Isotropic to CE Bias [pcm]: 1406.5\n",
+      "Angle to CE Bias [pcm]: 173.8\n"
+     ]
+    }
+   ],
+   "source": [
+    "print('Isotropic to CE Bias [pcm]: {0:1.1f}'.format(iso_bias))\n",
+    "print('Angle to CE Bias [pcm]: {0:1.1f}'.format(angle_bias))"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "We see a large reduction in error by switching to the usage of angle-dependent multi-group cross sections! \n",
+    "\n",
+    "Of course, this rodded and partially voided BWR problem was chosen specifically to exacerbate the angular variation of the reaction rates (and thus cross sections). Such improvements should not be expected in every case, especially if localized absorbers are not present.\n",
+    "\n",
+    "It is important to note that both eigenvalues can be improved by the application of finer geometric or energetic discretizations, but this shows that the angle discretization may be a factor for consideration.\n",
+    "\n",
+    "## Fission Rate Distribution Comparison\n",
+    "Next we will visualize the mesh tally results obtained from our three cases.\n",
+    "\n",
+    "This will be performed by first obtaining the one-group fission rate tally information from our state point files. After we have this information we will re-shape the data to match the original mesh laydown. We will then normalize, and finally create side-by-side plots of all."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 40,
+   "metadata": {
+    "collapsed": false,
+    "scrolled": f
```

Comment:
Instead of, or in addition to, can you plot the reaction rate errors between the two MG calculations and the "reference" CE calculation? This would better illustrate the points about the gradient in the isotropic case than the fission rates themselves. Of course this might not be very useful if we decide to reduce the number of histories simulated in these examples.